tests/lib/naming_test.py

```diff
@@ -169,5 +171,123 @@ def testParseServiceFile(self):
     filedefs._ParseFile(data, 'services')
     self.assertEqual(filedefs.GetService('HTTP'), ['80/tcp'])
 
+
+class Naming_Characterization_Tests(unittest.TestCase):
```

Comment:
NP, I'd added this when there were no tests.  I was wondering if you could think of any use for the effort here, if there was anything useful to salvage.  If not, we should scrap it, I'm ok to do that too.  Scrap it?